pinot-core/src/main/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPruner.java

```diff
@@ -60,8 +62,13 @@
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class ColumnValueSegmentPruner implements SegmentPruner {
 
+  private int _inPredicateThreshold;
+  public static final int DEFAULT_VALUE_FOR_IN_PREDICATE = 10;
```

Comment:
I think this can be moved to CommonConstants in inner Server class.

We should use `pinot.server.query.executor.pruner` prefix since that is what is used by QueryExecutorConfig when it creates SegmentPrunerConfig. Please check the QueryExecutorConfig constructor.

Secondly, since this config is applicable to min-max column value pruner on the server, we should use the convention `pinot.server.query.executor.pruner.columnvaluesegmentpruner.<your new in predicate config>`

This is because we create the config for each type of pruner as seen in the following code in SegmentPrunerConfig 

```
for (String segmentPrunerName : segmentPrunerNames) {
      _segmentPrunerNames.add(segmentPrunerName);
      _segmentPrunerConfigs.add(segmentPrunerConfig.subset(segmentPrunerName));
    }
```

When init() is called on ColumnValueSegmentPruner, it is initialized with all configs under `pinot.server.query.executor.pruner.columnvaluesegmentpruner.`

Reply:
Both config and default can be moved to CommonConstants